gkeys/gkeys/keyhandler.py

```diff
@@ -108,3 +111,27 @@ def key_search(self, args, first_match=False):
 
         self.logger.debug(_unicode("KeyHandler: key_search; keys = %s") % str(keys))
         return keys
+
+    def _is_expiring(self, results, days_limit):
+        is_expiring = False
+        for key in results:
+            for specs in results[key]:
+                if specs.days > days_limit*(-1) and specs.days < days_limit:
+                    is_expiring = True
+                    break
+        return is_expiring
```

Comment:
results variable name  here is not as meaningful.  keys would be a more suitable/meaningful variable name.